kokoro/check.sh

```diff
@@ -105,5 +105,17 @@ else
     echo Merge-base of $gtag and perry/master is $mtag
 fi
 
+#Check if faucet tags were added into forch
+for remote in faucet origin perry; do
+    faucet_tag=`git ls-remote $remote v1_0`
+    if [ -n "$faucet_tag" ]; then
+        echo
+        echo Error:
+        echo Faucet tag found in Forch repo: `git remote get-url $remote`
```

Comment:
this is redundant with the output at the top of the file for the remote URL... I'd keep it as the semantic name and not the url

Reply:
done, changed to semantic name